CIP-XXX/CIP-XXX.md

```diff
@@ -27,10 +27,53 @@ As part of this, we also describe the bit ordering within a `BuiltinByteString`,
 and provide some laws these operations should obey.
 
 ## Motivation: why is this CIP necessary?
+Currently, Plutus Core has a `BuiltinByteString` type, which is a packed array of bytes, similar to Haskell's ByteString.
```

Comment:
I prefer to leave a space between heading and body. I know this doesn't affect the result any, but I find it easier to read and edit that way.

Also, if we're going to be consistent, it should be `ByteString` in the same font as `BuiltinByteString`.

We can also drop 'Currently' here, as I doubt this will change any time ever.